test_plugin.py

```diff
@@ -72,34 +72,45 @@ class TestMessage(unittest.TestCase):
     def test_message_invert(self):
         test_cases = [
             plugin.Message(
+                vers="1.0",
```

Comment:
I think we can leave the version out of Message (or at least make it optional), since it's really the version of the serialization format and not the contents itself.

We can also make the defaults env the empty string "" and meta the empty {}.